Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -255,6 +255,17 @@
                         .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderString(roleIdToUserNamesMap[dataItem.Id][role.Id])))
             );
 
+            launcher = launcher
```

Comment:
Nogen grund til at den ligger for sig selv efter de andre?

Reply:
Ja den er afbrudt af loopet med rollerne. De skal placeres et bestem sted (åben for koden ovenfor denne blok) :-)